backend/src/database/repositories/automationRepository.ts

```diff
@@ -56,6 +64,17 @@ export default class AutomationRepository extends RepositoryBase<
 
     const transaction = this.transaction
 
+    const existingActiveAutomations = await this.findAndCountAll({
+      state: AutomationState.ACTIVE,
+    })
+
+    if (
+      data.state === AutomationState.ACTIVE &&
+      existingActiveAutomations.count >= PLAN_LIMITS[currentTenant.plan][FeatureFlag.AUTOMATIONS]
+    ) {
+      throw new Error(`Maximum number of active automations reached for the plan!`)
```

Comment:
why throw an error? shouldn't this be gracefully ignored?

Reply:
The error message helps the front end regarding the cause of operation failure, maybe we can catch this error and show the message to the user? also, do you think its bettter we ignore the error?

Reply:
ok - I thought it's used in the nodejs-worker when we actually process automation execution - there it would be pretty useless - just an error log amongst the many :)